src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -0,0 +1,36 @@
+---
+group: frontend-developer-guide
+title: Create a new layout
+functional_areas:
+  - Frontend
+---
+
+### Create a new page layout in custom theme
+
+create a custom page-layout xml file in the below directory.
+app/design/frontend/<VendorName>/<ThemeName>/Magento_Theme/page_layout/4columns.xml
+
+```xml
+<?xml version="1.0"?>
+<layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_layout.xsd">
+    <update handle="3columns"/>
+    <referenceContainer name="page.wrapper">
+        <container name="header.container" as="header_container" label="Page Header Container" htmlTag="header" htmlClass="page-header" before="main.content"/>
```

Comment:
We need to provide a better example of the additional container. If you refer to my previous comment with double footer, it might be "extra-footer" container or so.

Reply:
@rogyar I have updated the documentation as per your suggestion. Please review it.